### ISSUE TITLE
[a11y support] Platform Content Team: Internal Links Inquiry

### ISSUE THREAD

**Issue by sara-amanda** [department-of-veterans-affairs/va.gov-team#81833]
# CAIA a11y Support

- **Inquiry c/o Joe Knipes, product manager for the Platform Content Team**: [DSVA Slack](https://dsva.slack.com/archives/C01K37HRUAH/p1714136412599129)
    - **Note**: Joe will be OOO 4/29 and 4/30/2024.

## Inquiry

> - Platform Content Team would like to solicit some feedback from your team regarding whether internal links on [Platform Website ](https://depo-platform-documentation.scrollhelp.site/)should be updated to only open in the same tab.
> - In addition to user feedback, we are aware of [Web Content Accessibility Guidelines](https://www.w3.org/TR/WCAG20-TECHS/G200.html) surrounding this behavior.
> 

### Links

> All links should default to opening in the same tab.
>
> The only exception to this, is if the link is embedded in the content a form flow or task, and opening that link would cause the user to lose their place or data they've entered. This scenario should be avoided when possible. If the link does open in a new tab, the link label should clearly indicate that.
>
> See the [links section](https://design.va.gov/content-style-guide/links) in the content style guide for information on link labels.

[Source: GitHub Decision on links by Mikki](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1680#issuecomment-2013837291)

```[tasklist]
- [x] Covert Slack inquiry from DSVA to ticket.
- [x] Add Originator/Team name
- [x] Add Opened Date
- [x] Add Estimate
- [x] Add label(s) for discipline(s) involved in work
- [ ] Provide feedback
```

**Comment by sara-amanda** [comment]
@jknipes please find this ticket and feedback for your review. Let me know if you have any questions, or need additional information. Happy to help! 

cc: @coforma-terry

**Comment by jknipes** [comment]
@sara-amanda although the guidance is 
> All links should default to opening in the same tab.

in Scroll Viewport, the app that turns Confluence docs into Platform Website, **internal** links can be configured to open in either the same tab or a new tab. **External** links (to va.gov, GitHub, Mural, external websites) can **only** open in a new tab. Platform Content Team can begin work on the internal links, most likely during the next sprint.
